### ISSUE TITLE
Update to use latest Root libs?

### ISSUE THREAD

**Issue by ToysoftInc** [pedroSG94/Stream-USB-test#11]
Any chance of updating this lib?

**Comment by pedroSG94** [comment]
Hello,

It is difficult because I haven't a USB device to test now as you can see in Readme.
Also, the library that is used to connect with USB device has 5 years without updates so I can't do support for it and maybe is bugged in latest version of Android.
That's the 2 reasons because this library is unmaintained. 

If you can find a library to replace it with recent activity I can consider to update it but for now I'm with iOS version of the library so it will need to wait.

**Comment by ToysoftInc** [comment]
Yes the uvc lib is very old and I could not find any replacement.
